cmd/sharded-test-server/virtual.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2022 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/abiosoft/lineprefix"
+	"github.com/fatih/color"
+
+	"k8s.io/klog/v2"
+
+	"github.com/kcp-dev/kcp/cmd/test-server/helpers"
+	"github.com/kcp-dev/kcp/test/e2e/framework"
+)
+
+func startVirtual(ctx context.Context, logFilePath string) error {
+	klog.Infof("Starting virtual-workspaces standalone server")
+
+	prefix := "vw"
+	yellow := color.New(color.BgYellow, color.FgHiWhite).SprintFunc()
+	out := lineprefix.New(
+		lineprefix.Prefix(yellow(prefix)),
+		lineprefix.Color(color.New(color.FgHiYellow)),
+	)
+
+	commandLine := framework.DirectOrGoRunCommand("virtual-workspaces")
+	commandLine = append(
+		commandLine,
+		"--kubeconfig=.kcp-0/admin.kubeconfig",
+		"--context=system:admin",
```

Comment:
@s-urbaniak @csams @kylape reminder: we definitely don't want this medium term, but find a way, probably some special RBAC permission to do wildcard list/watch.